planet/specs.py

```diff
@@ -70,6 +70,19 @@ def validate_item_type(item_type):
     return _validate_field(item_type, supported_item_types, 'item_type')
 
 
+def validate_data_item_type(item_type):
+    '''Item types supported by the data api.'''
+    # This is a quick-fix for gh-956
+    return _validate_field(item_type, get_data_item_types(), 'item_type')
```

Comment:
@jreiberkyle since get_data_item_types() below is essentially a one line function, how about this instead? Let's avoid creating new functions when they're not strictly required at the moment.

```suggestion
    return _validate_field(item_type, get_item_types() | {'SkySatVideo'}, 'item_type')
```

Reply:
ohhh wait, I was going to say yes absolutely but the data CLI uses `get_data_item_types()` to list the valid item types in the help. So removing `get_data_item_types()` would require the same one-liner to be added to `planet/cli/data.py` as well. Introducing `get_date_item_types()` was chosen to keep the "hacktastic" code in one place within specs.

Reply:
but looking at the code, there are a lot of opportunities to get rid of short-term variables and extra lines of code. I did what I could while keeping `get_data_item_types()`